packages/rocketchat-api/server/v1/channels.js

```diff
@@ -370,6 +370,34 @@ RocketChat.API.v1.addRoute('channels.list.joined', { authRequired: true }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('channels.online', { authRequired: true }, {
```

Comment:
Can you add this to the `groups` as well? And if you're up for it, create the documentation on this over on our docs? https://github.com/RocketChat/Rocket.Chat.Docs/tree/master/6.%20Developer%20Guides/REST%20API